tests/e2e/support/commands.js

```diff
@@ -33,7 +33,6 @@ Cypress.Commands.add('loadModeler', () => {
 
   cy.viewport(defaultViewportDimensions.width, defaultViewportDimensions.height);
   cy.visit(url);
-  cy.reload();
```

Comment:
removed the cy.reload when hitting `/` since doesn't make any sense to reload the page on load